src/fmu/dataio/dataio.py

```diff
@@ -667,46 +677,37 @@ def _update_check_settings(self, newsettings: dict) -> None:
         self._validate_workflow_key()
         self._validate_and_establish_fmucontext()
 
-    def _establish_pwd_rootpath(self) -> None:
-        """Establish state variables pwd and the (initial) rootpath.
-
-        The self._pwd stores the process working directory, i.e. the folder
-        from which the process is ran
+    def _establish_rootpath(self) -> Path:
+        """Establish the rootpath. The rootpath is the folder that acts as the
+        base root for all relative output files. The rootpath is dependent on
+        if this is a FMU run and if it is run from inside/outside RMS:
 
-        The self._rootpath stores the folder from which is the base root for all
-        relative output files. This rootpath may be dependent on if this is a FMU run
-        or just an interactive run.
-
-        Hence this 'initial' rootpath can be updated later!
+        1: Running ERT: The rootpath will be equal to the casepath
+        2: Running RMS outside ERT: The rootpath will be rootpath/rms/model
+        3: Not running ERT or beeing inside RMS: The rootpath will be the pwd
```

Comment:
Difference point 2 and 3 not too clear for me, perhaps clarify?

Reply:
Tried to make it a bit more clear 👍 not sure if I managed though 😄